services/npms-io/score.service.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+const { coveragePercentage } = require('../color-formatters')
+
+// https://api-docs.npms.io/#api-Package-GetPackageInfo
+const numberSchema = Joi.number().min(0).max(1).required()
+const responseSchema = Joi.object({
+  score: Joi.object({
+    final: numberSchema,
+    detail: Joi.object({
+      maintenance: numberSchema,
+      popularity: numberSchema,
+      quality: numberSchema,
+    }),
+  }),
+}).required()
+
+const keywords = ['node', 'npm score']
+
+module.exports = class NpmsIO extends BaseJsonService {
```

Comment:
We need to make the file name and the service name match up, both for consistency and so that the test filters will work correctly.

Let's rename this file to `npmsio-score.service.js` (or `npms-io-score.servive.js`, your call), update the tester file name accordingly to match, and then rename the class too:

```suggestion
module.exports = class NpmsIOScore extends BaseJsonService {
```